projects/rulenode-core-config/src/lib/components/transform/copy-keys-config.component.html

```diff
@@ -1,28 +1,28 @@
 <section [formGroup]="copyKeysConfigForm" fxLayout="column">
-  <div class="mat-caption" style="margin-bottom: 8px">{{'tb.rulenode.copy-from' | translate}}</div>
-  <mat-radio-group formControlName="fromMetadata" fxLayout="row" fxLayoutGap="16px">
-    <mat-radio-button [value]="false" color="primary" >
-      <span>{{'tb.rulenode.data-to-metadata' | translate}}</span>
-    </mat-radio-button>
-    <mat-radio-button [value]="true" color="primary">
-      <span>{{'tb.rulenode.metadata-to-data' | translate}}</span>
-    </mat-radio-button>
-  </mat-radio-group>
-  <mat-form-field class="mat-block" subscriptSizing="dynamic">
-    <mat-chip-grid formControlName="keys" #attributeChipList>
-      <mat-chip-row
-        *ngFor="let key of copyKeysConfigForm.get('keys').value;"
-        (removed)="removeKey(key)">
-        {{key}}
-        <mat-icon matChipRemove>close</mat-icon>
-      </mat-chip-row>
-      <input matInput type="text" placeholder="{{ 'tb.rulenode.keys' | translate }}"
-             [matChipInputFor]="attributeChipList"
-             [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
-             (matChipInputTokenEnd)="addKey($event)"
-             [matChipInputAddOnBlur]="true">
-    </mat-chip-grid>
-    <mat-error *ngIf="copyKeysConfigForm.get('keys').hasError('required')">{{ 'tb.rulenode.keys-required' | translate }}</mat-error>
-    <mat-hint translate>tb.rulenode.use-regular-expression-hint</mat-hint>
-  </mat-form-field>
+  <div class="mat-caption pb-16" fxLayout="column">
+    <mat-form-field class="mat-block"  subscriptSizing="dynamic" >
+      <mat-label>{{ 'tb.rulenode.keys' | translate }}</mat-label>
+      <mat-chip-grid formControlName="keys" #attributeChipList >
```

Comment:
Could be replaced by tb-string-items-list.
If will be replaced all unused related mat-chip-grid could be deleted from controller (separatorKeysCodes, add key, removeKey...). Same for delete-keys component.